features/admin.applications.v1/api/application.ts

```diff
@@ -121,6 +121,36 @@ export const deleteApplication = (id: string): Promise<any> => {
         });
 };
 
+/**
+ * Deletes an application when the relevant id is passed in.
+ *
+ * @param id - ID of the application.
+ * @returns A promise containing the response.
+ */
+export const disableApplication = (id: string, status: boolean): Promise<any> => {
+    const requestConfig: AxiosRequestConfig = {
+        data: { "applicationEnabled":status },
+        headers: {
+            "Accept": "application/json",
+            "Access-Control-Allow-Origin": store.getState().config.deployment.clientHost,
+            "Content-Type": "application/json"
+        },
+        method: HttpMethods.PATCH,
+        url: store.getState().config.endpoints.applications + "/" + id
```

Comment:
Use `ES6` template literals.

```suggestion
        url: `${ store.getState().config.endpoints.applications }/${  id }`
```